### ISSUE TITLE
[central ui] Need a link to Katib UI

### ISSUE THREAD

**Issue by jlewi** [kubeflow/katib#90]
central-ui should provide a link to Katib.

This is blocked on a ksonnet component for Katib (#32)

Since Katib isn't always deployed it would be nice if the central UI could convey whether or not Katib was running (e.g. greying out the link if Katib isn't available).

/priority p1
/cc @swiftdiaries

**Comment by swiftdiaries** [comment]
Will take this up.

**Comment by jlewi** [comment]
@swiftdiaries #118 is fixed.

**Comment by swiftdiaries** [comment]
Thanks @jlewi, I'll get this in.

**Comment by jlewi** [comment]
Duplicate of #128
